src/index.css

```diff
@@ -50,4 +50,12 @@ h5 {
 
 input[type = 'text'] {
   @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+}
+
+textarea[type = 'textarea'] {
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+}
+
+select{
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
```

Comment:
This seems to be broken on safari, does it work on yours?

Reply:
The styling works on chrome for me.

Reply:
Try this https://stackoverflow.com/questions/7638677/how-can-i-remove-the-gloss-on-a-select-element-in-safari-on-mac